content/plugins/dll-plugin.md

```diff
@@ -0,0 +1,148 @@
+---
+title: DllPlugin
+contributors:
+  - aretecode
+  - sokra
+  - opiepj
+  - simon04
+---
+
+## Introduction
+
+The `DllPlugin` and `DllReferencePlugin` provide means to split bundles in a way that can drastically improve build time performance.
+
+[DllPlugin](#DllPlugin) is used in a separate webpack config exclusively to create a dll-only-bundle. It creates a `manifest.json` file, which is used by the [DllReferencePlugin](#DllReferencePlugin) to map dependencies.
+
+[DllReferencePlugin](#DllReferencePlugin) is used in the primary webpack config, it references the dll-only-bundle(s) to require pre-built dependencies.
+
+
+
+## `DllPlugin`
+
+* `context` (optional): context of requests in the manifest file (defaults to the webpack context.)
+* `name`: name of the exposed dll function ([TemplatePaths][src-TemplatedPathPlugin]: `[hash]` & `[name]` )
+* `path`: **absolute path** to the manifest json file (output)
+
+```javascript
+new webpack.DllPlugin(options)
+```
+
+Creates a `manifest.json` which is written to the given `path`. It contains mappings from require and import requests, to module ids. It is used by the [DllReferencePlugin](#DllReferencePlugin).
+
+
+W> Keep the `name` consistent with `output.library`.
+
+Combine this plugin with `output.library` option to expose (aka, put into the global scope) the dll function.
+
+
+### `DllReferencePlugin`
+
+* `context`: (**absolute path**) context of requests in the manifest (or content property)
+* `content` (optional): the mappings from request to module id (defaults to `manifest.content`)
+* `manifest` (object): an object containing `content` and `name`
```

Comment:
I see the alphabetical ordering. Still, since `content` refers to `manifest`, put `manifest` before `content`?